### ISSUE TITLE
FAILURE: Build failed with an exception

### ISSUE THREAD

**Issue by hungntv** [nccgroup/readable-thrift#2]
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Volumes/Backup/Projects/readable-thrift/build.gradle' line: 72

* What went wrong:
A problem occurred evaluating root project 'readablethrift'.
> Could not get unknown property 'sonatypeUsername' for object of type org.gradle.api.publication.maven.internal.deployer.DefaultGroovyMavenDeployer.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 6s
```

How to fix it?

**Comment by hungntv** [comment]
The build scan: https://gradle.com/s/za5sh4ouugjiu

**Comment by jamchamb** [comment]
Looks like there is a fix for this in issue #1.
